src/app/views/cheat-sheets/science/science.component.html

```diff
@@ -135,9 +135,9 @@
     </div>
     <p class="text-muted text-center">Use the formula above to calculate how many labs satisfy a target consumption rate of science packs per minute.</p>
     <p class="text-muted">
-        Research Cycle Time = Time in seconds to complete one research cycle (usually 60 seconds for all infinite tech)<br>
+        Research Cycle Time = Time in seconds to complete one research cycle, this is usually 60 seconds for all infinite tech.<br>
```

Comment:
lets make the definitions bold too i think
```suggestion
        <b>Research Cycle Time</b> - Time in seconds to complete one research cycle, this is usually 60 seconds for all infinite tech.<br>
```